spec/spec_helper.rb

```diff
@@ -1,14 +1,9 @@
-require 'inherited_class_var'
-require 'bundler/setup'
-require 'coveralls'
-
-begin
-  require 'pry'
-rescue LoadError
-end
-
+Bundler.require(:default, :test)
 Coveralls.wear!
 
+$LOAD_PATH.unshift File.expand_path('../../lib', __FILE__)
+require 'inherited_class_var'
+
 RSpec.configure do |c|
   c.run_all_when_everything_filtered = true
-end
+end
```

Comment:
slightly cleaner. did you figure out coveralls? it took me too long to figure it out for `csv_row_model`